test/helpers.py

```diff
@@ -0,0 +1,32 @@
+from betamax.cassette.cassette import Placeholder
+
+CLASSIFIED_COOKIES = ('gspkauth',)
+
+
+def sanitize_cookies(interaction, cassette):
+    response = interaction.as_response()
+    response_cookies = response.headers.get('Set-Cookie', None)
```

Comment:
I think you can radically simplify this method by using [Request and Response cookies attribute](http://docs.python-requests.org/en/master/api/#requests.Response.cookies), instead of parsing all the headers manually.

Reply:
Fixed in 3c5514f2c920ed8297caef0a08ede09bebbe5b2b (not yet pushed). I still had to do some parsing because 

1. Some new secret values are sent in request bodies, not headers
2. The request objects provided don't have a `cookies` attribute even though the responses do.